lambda_functions/analyzer/yara_analyzer.py

```diff
@@ -92,6 +98,49 @@ def _yara_variables(original_target_path: str) -> Dict[str, str]:
             'filetype': file_suffix.upper()  # Used in only one rule (checking for "GIF").
         }
 
+    def _yextend_matches(self, target_file: str) -> List[YaraMatch]:
+        """Use yextend to check for YARA matches against archive contents.
+
+        Args:
+            target_file: Local path to target file to be analyzed.
+
+        Returns:
+            List of YaraMatch tuples, or an empty list if yextend didn't work correctly.
+        """
+        try:
+            output = subprocess.check_output(
+                ['./yextend', '-r', self._compiled_rules_file, '-t', target_file, '-j'],
+                stderr=subprocess.STDOUT
+            )
+        except subprocess.CalledProcessError:
+            LOGGER.exception('Yextend invocation failed')
+            return []
+
+        try:
+            decoded_output = output.decode('utf-8')
+        except UnicodeDecodeError:
+            LOGGER.error('Yextend output could not be decoded to utf-8:\n%s', output)
```

Comment:
Do you consider to use `LOGGER.exception` here?

Reply:
I did consider it, but to me the traceback here wasn't adding any useful information. We already know exactly what the exception was, and the output is all we need for debugging. This declutters the logs a bit